client/controllers/incidentReports/incidentModal.coffee

```diff
@@ -68,13 +67,12 @@ Template.incidentModal.events
         if not error
           $('.reactive-table tr').removeClass('open')
           $('.reactive-table tr.tr-details').remove()
-          if !duplicate
-            form.reset()
-            notify('success', 'Incident added.')
-            if instance.data.offCanvas
-              stageModals(instance, instance.modals)
-            else
-              Modal.hide('incidentModal')
+          form.reset()
```

Comment:
I wonder if the reset is necessary considering the form is completely removed from the DOM after the modal is dismissed?